monitoring/tracing/trace/span.go

```diff
@@ -3,95 +3,68 @@ package trace
 import (
 	"context"
 
-	"go.opencensus.io/trace"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
+	"go.opentelemetry.io/otel/trace/noop"
 )
 
-// TracingEnabled tracks whether tracing is enabled in prysm.
+// TracingEnabled tracks whether tracing is enabled in your application.
 var TracingEnabled = false
 
-// StartSpan is a wrapper over the opencensus package method. This is to allow us to skip
+// StartSpan is a wrapper over the OpenTelemetry package method. This is to allow us to skip
 // calling that particular method if tracing has been disabled.
-func StartSpan(ctx context.Context, name string, o ...trace.StartOption) (context.Context, *trace.Span) {
+func StartSpan(ctx context.Context, name string, opts ...trace.SpanStartOption) (context.Context, trace.Span) {
 	if !TracingEnabled {
 		// Return an empty span if tracing has been disabled.
-		return ctx, trace.NewSpan(EmptySpan{})
+		return ctx, noop.Span{}
 	}
-	return trace.StartSpan(ctx, name, o...)
+	tracer := otel.Tracer("otel-tracer")
```

Comment:
```suggestion
	tracer := otel.Tracer("")
```
If you use empty string, it should load the default name from the global tracer. I suspect that would be the service name and that's what we want instead of a very generically named `otel-tracer`